CRAYSAT-1624: Document errors and warnings in bos-operations stage (#2838)
Cray-HPE/docs-csm: operations/power_management/Power_On_and_Boot_Compute_Nodes_and_User_Access_Nodes.md

    All BOS sessions completed.
    ```

>     Note the returned job ID for each session; for example: `"boa-79584ffe-104c-4766-b584-06c5a3a60996"`

Reviewer: ```suggestion
    Note the returned job ID for each session, for example: `"boa-79584ffe-104c-4766-b584-06c5a3a60996"`.
```

    **Note:** In certain cases, the command may display an error similar to the following:

    ```text
    ERROR: Failed to get state of nodes in session template 'UAN_SESSION_TEMPLATE': Failed to get state of nodes with role=['Application', 'Application_UAN'] for boot set 'BOOT_SET' of session template 'UAN_SESSION_TEMPLATE': GET request to URL 'https://api-gw-service-nmn.local/apis/smd/hsm/v2/State/Components' failed with status code 400: Bad Request. Bad Request Detail: bad query param: Argument was not a valid HMS Role
    ```

>     This error is a non-fatal error and does not affect the `bos-operations` stage of `sat bootsys`.

Reviewer: ```suggestion
    This is a non-fatal error and does not affect the `bos-operations` stage of `sat bootsys`.
```

    **Note:** In certain cases, the command may fail before reaching the displayed timeout

>     and will show warnings similar to the following:

Reviewer: ```suggestion
    and show warnings similar to the following:
```

    ```text
    WARNING: The 'kubectl wait' command failed instead of timing out. stderr: error: condition not met for jobs/boa-79584ffe-104c-4766-b584-06c5a3a60996
    ```

>     The existence of these warnings does not prevent the BOS operation from proceeding, however it
>     may result in the `bos-operations` stage of the `sat bootsys` command exiting before the BOS
>     operation is complete. In this case it is especially important to view the logs to monitor the
>     boot, and to verify that the nodes reached the expected state, both of which are shown
>     in the remaining steps.

Reviewer: ```suggestion
    The BOS operation can still proceed even with these warnings. However, the warnings
    may result in the `bos-operations` stage of the `sat bootsys` command exiting before the BOS
    operation is complete. Because of this, it is important to view the logs in order to monitor the
    boot and to verify that the nodes reached the expected state. Both of these recommendations are shown
    in the remaining steps.
```
